pages/index.jsx

```diff
@@ -5,15 +5,48 @@ import Image from 'next/image';
 import Link from 'next/link';
 import Head from 'next/head';
 
+export const metadata = {
+  title: "Librelinks",
+  description: "A website where essential tools, prompts, and datasets are thoughtfully curated and combined in one place for your convenience.",
```

Comment:
The description of the project in the metadata file does not hold for Librelinks. I think you have mistaken It with another project.

Reply:
Yes, I bad. Wait, I will fix it.